route/build/src/standalone/read_runoff.f90

```diff
@@ -353,6 +353,9 @@ subroutine read_1D_runoff(fname,          &  ! input: filename
  ! replace _fill_value with -999 for dummy
  where ( abs(dummy - input_fillvalue) < verySmall ) dummy = realMissing
 
+ ! replacing the negative values
```

Comment:
if checking negative value, why not where (dummy < 0._dp) dummy = 0._dp?

Reply:
I have commented them at this moment. we do this in sort now sth like ```where(sorted_flux <0._dp) sorted_flux = 0._dp```